### DISCUSSION TITLE
Add more example

### DISCUSSION THREAD

**Discussion by richerfu** [Tencent/tquic#203] [Ideas（新功能的建议和讨论）]
Add more example , for example: h3 request etc.

**Comment by glime-ai** [comment]
To add more examples, such as an HTTP/3 (h3) request, you can follow the structure provided in the feature request template on the TQuic GitHub repository. Here's a structured way to suggest adding more examples, specifically for HTTP/3 requests:

---

**Is your feature request related to a problem? Please describe.**
Yes, the current documentation and examples are helpful, but they lack specific examples for HTTP/3 requests. This makes it difficult for new users to understand how to implement or use HTTP/3 features with TQuic.

**Describe the solution you'd like**
I would like to see more examples added to the documentation, specifically examples that demonstrate how to make HTTP/3 requests using TQuic. These examples should cover common use cases and highlight best practices.

**Describe alternatives you've considered**
While the existing documentation provides a good foundation, and external resources offer some insights into HTTP/3, having dedicated examples within the TQuic documentation would streamline the learning process and provide a single source of truth.

**Additional context**
HTTP/3 is becoming increasingly important for web performance and security. Providing clear, concise, and practical examples of HTTP/3 requests would greatly benefit TQuic users and contribute to wider adoption and understanding of both TQuic and HTTP/3.

---

You can submit this feature request by following the guidelines on the TQuic GitHub repository's feature request template. Make sure to provide as much detail as possible to help the developers understand the need and potential impact of your suggestion.

**Comment by iyangsj** [comment]
For more examples, see https://github.com/Tencent/tquic/tree/develop/tools
